builtin/providers/aws/config.go

```diff
@@ -213,6 +215,9 @@ func (c *Config) Client() (interface{}, error) {
 
 		log.Println("[INFO] Initializing CodeDeploy Connection")
 		client.codedeployconn = codedeploy.New(sess)
+
+		log.Println("[INFO] Initializing CodeCommit SDK connection")
+		client.codecommitconn = codecommit.New(awsConfig)
```

Comment:
you are 100% correct. I have just updated this commit. It should indeed be the sess work

I'm also surprised this built